src/Discord.Net.Rest/Entities/Users/RestUser.cs

```diff
@@ -65,6 +65,13 @@ internal virtual void Update(Model model)
             Update(model);
         }
 
+        /// <summary>
+        ///     Returns a direct message channel to this user, or create one if it does not already exist.
+        /// </summary>
+        /// <param name="options">The options to be used when sending the request.</param>
+        /// <returns>
+        ///     An awaitable <see cref="Task"/> containing a <see cref="RestDMChannel"/>.
```

Comment:
Follow the new guideline we started following since https://github.com/Still34/Discord.Net/commit/b940ecb95e221659c2af920327d692aaf654bceb based on `docs.microsoft.com`